helm-charts/vmpooler/values.yaml

```diff
@@ -18,7 +18,7 @@ timeout: '15'
 
 redis_connection_pool_size: '200'
 redis_connection_pool_timeout: '40'
-redis_reconnect_attempts: '40'
+redis_reconnect_attempts: '[0,0.25,0.5,1,2,4,8,16,32,64,128,256,512]'
```

Comment:
I think this change as well as updating the vmpooler gem should happen in https://github.com/puppetlabs/vmpooler-deployment/pull/114 first